conans/client/cmake.py

```diff
@@ -156,3 +165,48 @@ def runtime(self):
         if runtime:
             return "-DCONAN_LINK_RUNTIME=/%s" % runtime
         return ""
+
+    def configure(self, conan_file, args=None, vars=None, source_dir=None, build_dir=None):
```

Comment:
Hehe :) Yes length of `variables` is borderline, it was one of my alternatives.
From CMake documentation:

> `-D <var>:<type>=<value>`
>
>    Create a cmake cache entry.
>
>    When cmake is first run in an empty build tree, it creates a CMakeCache.txt file and populates it with customizable settings for the project. This option may be used to specify a setting that takes priority over the project’s default value. The option may be repeated for as many cache entries as desired.

So they refer to a CMake variable as both `var`, `cache entry` and `setting`, making this even more confusing. The conan documentation should make it really clear what is happening here, regardless of which name we choose.

Also thanks for creating the issue!